File: Linguagem C++/Vetores/Negativos.cpp
#include <iostream>

using namespace std;

int main()
{
    int n;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> n;

    int vet[n];

    //Lendo o vetor
    for (int i=0;i<=n-1;i++){
        cout << "Digite um numero: ";
        cin >> vet[i];
    };

    //Imprimindo os numeros negativos
    cout << "NUMEROS NEGATIVOS:" << endl;
    for (int i=0;i<=n-1;i++){
        if (vet[i]<0){
            cout << vet[i] << endl;
        };
    };

    return 0;
}

File: Linguagem Java/Exercicios/Negativos.java
import java.util.Locale;
import java.util.Scanner;

public class Negativos {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n;
		
		System.out.print("Quantos numeros voce vai digitar: ");
        n = sc.nextInt();
        
        int vet[] = new int[n];
        
        // Lendo o vetor
        for (int i=0;i<n;i++) {
        	System.out.print("Digite um numero: ");
        	vet[i] = sc.nextInt();
        };
        
        // Mostrando os numeros negativos
        System.out.println("NUMEROS NEGATIVOS:");
        for (int i=0;i<n;i++) {
        	if (vet[i] < 0) {
        		System.out.println(vet[i]);        		
        	};        	
        };
        
        sc.close();
	}

}
